examples/nm.rs

```diff
@@ -1,8 +1,6 @@
-extern crate memmap;
-extern crate object;
-
 use std::{env, fs, process};
 
+use memmap;
```

Comment:
This is not necessary, because other crates get added to the prelude in the 2018.

Reply:
Thanks for spotting. I didn't know that before.